src/libraries/Common/src/Interop/Linux/OpenLdap/Interop.Ldap.cs

```diff
@@ -77,6 +77,9 @@ static Ldap()
         [DllImport(Libraries.OpenLdap, EntryPoint = "ldap_initialize", CharSet = CharSet.Ansi, SetLastError = true)]
         public static extern int ldap_initialize(out IntPtr ld, string hostname);
 
+        [DllImport(Libraries.OpenLdap, EntryPoint = "ldap_create", CharSet = CharSet.Ansi, SetLastError = true)]
+        public static extern int ldap_create(out IntPtr ld);
+
         [DllImport(Libraries.OpenLdap, EntryPoint = "ldap_init", CharSet = CharSet.Ansi, SetLastError = true)]
         public static extern IntPtr ldap_init(string hostName, int portNumber);
```

Comment:
this is now dead? (and the others you replaced?)

Reply:
Do you mean `ldap_init` and `ldap_simple_bind`? Yes, but I avoided removing them because they were marked as public. I now see that it's in an internal class though—does that mean they're safe to delete?

Reply:
Yes. (BTW, if you make a breaking API change, we have validation steps that should error out the build.)